### PR TITLE
[fdk-aac] Update to version 2.0.2.

### PR DESCRIPTION
**Pull request by Sibras**
Updated fdk to use latest tagged version.
Converted to using fdk's internal cmake file

### CODE REVIEW

File: ports/fdk-aac/portfile.cmake [hunk 1]
```diff
@@ -3,21 +3,21 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
...
+    OPTIONS_RELEASE -DBUILD_PROGRAMS=OFF
 )
 
 vcpkg_install_cmake()
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/${PORT})
+vcpkg_fixup_pkgconfig()
 vcpkg_copy_pdbs()
-vcpkg_copy_tool_dependencies(${CURRENT_PACKAGES_DIR}/tools/fdk-aac)
+vcpkg_copy_tool_dependencies(${CURRENT_PACKAGES_DIR}/tools/${PORT})
 file(INSTALL ${SOURCE_PATH}/NOTICE DESTINATION ${CURRENT_PACKAGES_DIR}/share/fdk-aac RENAME copyright)
```
Comment (JonLiu1993) [inline ports/fdk-aac/portfile.cmake:22]:
```suggestion
file(INSTALL ${SOURCE_PATH}/NOTICE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```

File: ports/fdk-aac/portfile.cmake [hunk 2]
```diff
@@ -3,21 +3,21 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 vcpkg_from_github(
```
Comment (JonLiu1993) [inline ports/fdk-aac/portfile.cmake:3]:
```suggestion
vcpkg_fail_port_install(ON_ARCH "arm"  ON_TARGET "uwp")
vcpkg_from_github(
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/fdk-aac/portfile.cmake (hunk 2 above)
Comment (Sibras) [reply ports/fdk-aac/portfile.cmake:3]: The Ci passed on arm-uwp so im not sure what the issue is, am i missing something?

Comment (Sibras) [reply ports/fdk-aac/portfile.cmake:3]: Nevermind, I missed the supports field in the control file.

Comment (Sibras) [reply ports/fdk-aac/portfile.cmake:3]: The new cmake build file seems to support arm and uwp so this is no longer required

File: ports/fdk-aac/vcpkg.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "fdk-aac",
+  "version-string": "2.0.2",
```
Comment (JonLiu1993) [inline ports/fdk-aac/vcpkg.json:3]: We should use "version-semver" instesd of "version-string"

File: versions/f-/fdk-aac.json [hunk 4]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "c5735b2f5cd0fd265adf4749b1e87d97df75b48c",
+      "version-string": "2.0.2",
```
Comment (JonLiu1993) [inline versions/f-/fdk-aac.json:5]: We should use "version-semver" instesd of "version-string"

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/fdk-aac/vcpkg.json (hunk 3 above)
Comment (Sibras) [reply ports/fdk-aac/vcpkg.json]: done

File: versions/f-/fdk-aac.json (hunk 4 above)
Comment (Sibras) [reply versions/f-/fdk-aac.json]: done

**Review by JonLiu1993 — APPROVED** [review]

**Comment by ras0219-msft** [comment]
LGTM, thanks for the fix & update!

**Comment by cenit** [comment]
can you please add the feature as a default one in vcpkg-ci-ffmpeg also for these new valid configs, so that we test it automatically?

**Pull request merged**